README.md

```diff
@@ -233,3 +233,7 @@ When fetching public keys from MAuth, the following rules apply:
 During development classes are typically not cached in Rails applications.
 If this is the case, be aware that the MAuth-Client middleware object will be instantiated anew for each request;
 this will cause applications performing local authentication to fetch public keys before each request is authenticated.
+
+## Protocol Versions
+
+The mauth V2 protocol was added as of v5.0.0. This protocol updates the string_to_sign to include query parameters, uses different authentication header names, and has a few other changes. See this document for more information: (DOC?). By default mauth client will sign requests with both V1 and V2, sign responses with only the version used in the request, authenticate requests with only the highest version of the protocol present, and authenticate responses with only the highest version of the protocol present.
```

Comment:
Is there (or will there be) a confluence doc detailing v2?

Reply:
I think @ChazzMaTazz has already something